src/main/osd/osd.c

```diff
@@ -92,6 +92,12 @@
 #include "hardware_revision.h"
 #endif
 
+#ifdef USE_DSHOT_TELEMETRY
```

Comment:
These includes should not be conditional - `USE_HARDWARE_REVISION_DETECTION` above is an exception because it is legacy code.